app/src/main/java/com/example/chattutorial/ChannelActivity3.kt

```diff
@@ -45,35 +43,21 @@ class ChannelActivity3 : AppCompatActivity(R.layout.activity_channel_3) {
                 }
             }
 
-        val channelController = ChatClient.instance().channel(cid)
-        val currentlyTyping = MutableLiveData<Set<String>>(emptySet())
-
-        channelController.subscribe {
-            val typing = currentlyTyping.value ?: emptySet()
-            val typingCopy: MutableSet<String> = typing.toMutableSet()
-            when (it) {
-                is TypingStartEvent -> {
-                    val name = it.user.extraData["name"] as String
-                    typingCopy += name
-                    currentlyTyping.postValue(typingCopy)
+        channelHeaderView.text = "nobody is typing"
```

Comment:
We should just call `updateChannelHeaderView`, since we have it available. We can do it after creating `currentlyTyping`, or pass in an explicit `emptySet()`